collection-scripts/gather_network_logs_basics

```diff
@@ -19,8 +19,9 @@ function gather_multus_data {
   done
 }
 
-CLUSTER_NODES="${@:-$(oc get node -l node-role.kubernetes.io/master -oname)}"
-/usr/bin/gather_multus_logs $CLUSTER_NODES
+READY_CONTROL_PLANE_NODES="${@:-$(oc get node -l node-role.kubernetes.io/control-plane -o json | jq -r '.items[] | select(.status.conditions[] | select(.type=="Ready" and .status=="True")).metadata.name')}"
```

Comment:
We may want to do this filtering with just oc (to reduce dependencies on jq). 
That said jq is used in other parts of must-gather - https://github.com/openshift/must-gather/blob/3a90e581047a71a74fd9babb1ada3ad5f54c7a25/collection-scripts/gather_apirequestcounts#L11 

cc: @soltysh

Reply:
@sferich888 unfortunately, `jsonpath` nor `template` can list the nodes based on their readiness. I was working on something similar - using the `jq` is way simpler and better.